frontend/javascripts/viewer/controller/combinations/tool_controls.ts

```diff
@@ -583,30 +587,59 @@ export class EraseToolController {
 
   static onToolDeselected() {}
 }
-export class PickCellToolController {
+export class VoxelPipetteToolController {
   static getPlaneMouseControls(_planeId: OrthoView): any {
     return {
-      leftClick: (pos: Point2, _plane: OrthoView, _event: MouseEvent) => {
-        VolumeHandlers.handlePickCell(pos);
+      mouseMove: (
+        _delta: Point2,
+        position: Point2,
+        plane: OrthoView | null | undefined,
+        event: MouseEvent,
+      ) => {
+        MoveHandlers.moveWhenAltIsPressed(_delta, position, plane, event);
+      },
+      leftClick: (position: Point2, plane: OrthoView, event: MouseEvent) => {
+        if (event.shiftKey) {
+          VolumeHandlers.handlePickCell(position);
+          return;
+        }
+
+        const state = Store.getState();
+        const lastMeasuredGlobalPosition =
+          state.uiInformation.measurementToolInfo.lastMeasuredPosition;
+
+        if (lastMeasuredGlobalPosition == null) {
+          const globalPosition = calculateGlobalPos(state, position, plane).floating;
+          Store.dispatch(setLastMeasuredPositionAction(globalPosition));
+        } else {
+          Store.dispatch(hideMeasurementTooltipAction());
+        }
+        Store.dispatch(setIsMeasuringAction(true));
       },
     };
   }
 
+  static onToolDeselected() {
+    Store.dispatch(hideMeasurementTooltipAction());
+  }
+
   static getActionDescriptors(
     _activeTool: AnnotationTool,
     _userConfiguration: UserConfiguration,
-    _shiftKey: boolean,
+    shiftKey: boolean,
     _ctrlOrMetaKey: boolean,
     _altKey: boolean,
     _isTDViewportActive: boolean,
+    _activeToolWithoutModifiers: AnnotationTool,
   ): ActionDescriptor {
     return {
-      leftClick: "Pick Segment",
+      leftClick:
+        _activeToolWithoutModifiers === AnnotationTool.VOXEL_PIPETTE && !shiftKey
```

Comment:
Without checking the code too deeply: 
Isn't it ensured that once this function is called `AnnotationTool.VOXEL_PIPETTE` is always the active tool? Thus, this check might not be needed :thinking:

Reply:
there is a difference between the "adapted tool" and "active tool" (e.g., when being in the brush tool and then holding shift, the active tool is adapted to the pipette). originally, I wanted to achieve that "Pin Tooltip" is not shown when pressing shift in the brush tool (because I wanted to old pipette behavior that can simply select a segment id on shift+click). afterwards, I also ensured that the same behavior is achieved when being in the **pipette** tool and holding shift. the shift-condition made the condition on activeToolWithoutModifiers obsolete now, which is why I removed that code again.